scss/util/_breakpoint.scss

```diff
@@ -77,7 +77,7 @@ $breakpoint-classes: (small medium large) !default;
   // Convert any pixel, rem, or unitless value to em
   $bp: -zf-bp-to-em($bp);
   @if $bp-max {
-    $bp-max: -zf-bp-to-em($bp-max) - (1/16);
+    $bp-max: -zf-bp-to-em($bp-max) - 0.0001;
```

Comment:
Why `0.0001` ?

Side note, numbers < 1 are usually written without the first `0` in CSS/Sass. Like `.0001`

Reply:
Since all the `em` math works in 16ths, values end up at 4 decimals. I can remove the leading zero. Do you have a suggestion other than `.0001`?